code/modules/medical/surgery_procs.dm

```diff
@@ -100,7 +100,7 @@ var/global/list/chestitem_whitelist = list(/obj/item/gnomechompski, /obj/item/gn
 		surgeon, "<span class='alert'>You make a [fluff]cut into [patient]'s [target_area] with [src]!</span>")
 
 		patient.TakeDamage(surgeon.zone_sel.selecting, damage, 0)
-		take_bleeding_damage(patient, surgeon, damage)
+		take_bleeding_damage(patient, surgeon, damage, surgery_bleed = 1)
```

Comment:
on second thought, fixing surgical fuckups should be an active thing, not passive
```suggestion
		take_bleeding_damage(patient, surgeon, damage)
```

Reply:
^